includes/discovery/sensors/state/ray.inc.php

```diff
@@ -2,11 +2,11 @@
 
 echo 'RAy Racom State';
 
-// System Status (Value : na (0) unknown, ok (1) ok, warning (2) warning, alarm (3) alarm)
-$state = snmp_get($device, "systemStatus.0", "-Ovqe", 'RAY-MIB');
-if ($state) {
+// System Status Ray1 and Ray2 (Value : na (0) unknown, ok (1) ok, warning (2) warning, alarm (3) alarm)
+$state = snmp_get($device, "systemStatus", "-Ovqe", 'RAY-MIB');
+if (is_numeric($state)) {
     //Create State Index
-    $state_name = 'systemStatus';
+    $state_name = 'systemStatus-old';
```

Comment:
Do not rename existing state sensors.

Also, instead of doing two gets you could just do:
```
$state = snmp_getnext($device, "systemStatus", "-Ovqe", 'RAY-MIB');
```

Reply:
well and how do I make sure `'.1.3.6.1.4.1.33555.1.1.3.1'` or ` '.1.3.6.1.4.1.33555.1.1.3.1.0'` is used in discover_sensor (), depending on which device is detected?

Reply:
if you change the snmp output options, you can get the oid back too.